pom.xml

```diff
@@ -86,6 +86,11 @@
             <!-- TODO change from SNAPSHOT to v0.9.0 -->
             <version>0.9.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.wavefront</groupId>
+            <artifactId>wavefront-opentracing-sdk-java</artifactId>
```

Comment:
remove the opentracing-api dependency if you are adding this.

Reply:
Need it for `WavefrontTracer`, `WavefrontSpan`, etc.

Reply:
lets remove opentracing-api as it is a transitive dependency from opentracing sdk
```
<dependency>
             <groupId>io.opentracing</groupId>
             <artifactId>opentracing-api</artifactId>
             <version>0.31.0</version>
             <scope>provided</scope>
         </dependency>
```